internal/c8dutil/memory_label_store.go

```diff
@@ -0,0 +1,58 @@
+package c8dutil
```

Comment:
Perhaps make this its own package to prevent `c8dutil` from becoming too much of a grab-bag of things (with many imports); given that this is a test-util, perhaps `internal/testutils/labelstore` (or `internal/testutils/memorylabelstore`, but that's a bit lengthy)